{{cookiecutter.project_shortname}}/scripts/update.sh

```diff
@@ -2,7 +2,7 @@
 {% include 'misc/header.py' %}
 set -e
 
-./bootstrap
+./bootstrap.sh
```

Comment:
The idea with the scripts ``bootstrap`` and friends was to align it with https://github.com/github/scripts-to-rule-them-all thus not adding ``.sh`` suffix.